### PR TITLE
Reset and update password created

### PR DESCRIPTION
**Pull request by yviswanadh-innoclique**
Please review my changes

### CODE REVIEW

File: DataServices/UserService.js [hunk 1]
```diff
@@ -132,6 +134,43 @@ catch(err)
...
+        User.save();
+
+        return { status: "temp psw email sent" };
+    }
+
+
+}
+
+exports.updatePassword = async (resetModel) => {
+    id = resetModel.userId;
```
Comment (innoclique) [inline DataServices/UserService.js:170]: what is user not found 
we need to handle all else cases

File: Routes/IdentityRoute.js [hunk 2]
```diff
@@ -16,6 +16,8 @@ router.post("/GetUserByPhoneNumber", IdentityRoute.FindUserByPhoneNumber);
 router.post("/CreateAccount", IdentityRoute.CreateAccount);
 router.post("/CreateStaffAccount", IdentityRoute.CreateStaffAccount);
 router.post("/Authenticate", IdentityRoute.Authenticate);
+router.post("/updatePassword", IdentityRoute.updatePassword);
```
Comment (innoclique) [inline Routes/IdentityRoute.js]: follow case style through out app

**Review by innoclique — CHANGES_REQUESTED** [review]
do commented changes and recommit.

File: Routes/IdentityRoute.js (hunk 2 above)
Comment (yviswanadh-innoclique) [reply Routes/IdentityRoute.js]: Updated as per your suggestion

**Review by innoclique — APPROVED** [review]
approving

**Pull request merged**